main.tf

```diff
@@ -9,7 +9,7 @@ module "default_label" {
 }
 
 module "notify_slack" {
-  source               = "git::https://github.com/terraform-aws-modules/terraform-aws-notify-slack?ref=tags/v2.3.0"
+  source               = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.17.0"
```

Comment:
@osterman Looks like something went wrong here.

Reply:
that's a wierd one! =) thanks for catching that.

Reply:
wonder if it's because it's an external module `terraform-aws-modules/terraform-aws-notify-slack`